### ISSUE TITLE
Feature Request - check 404 page

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#7333]
**Is your feature request related to a problem? Please describe.**

https://github.com/ArctosDB/arctos/issues/6962 seems to be destined to involve "lots" of 404s as the media pages are removed. Can we somehow make that less-evil?

**Describe what you're trying to accomplish** 

Not look like Arctos is broken when things are deleted.

**Describe the solution you'd like**

???????????????????????

**Describe alternatives you've considered**

more ???????????????????????

**Additional context**

https://arctos.database.museum/media/999999999999 is inexpensive but not very useful.

"Arctos" probably will not know why Media URIs are broken, a redirect would be up to the collections.

**Priority**

Still ???????????????????????

**Comment by Jegelewicz** [comment]
I am not getting something about redirects - it seems like the situation described at DMNS is exactly what a redirect is for? I have a photo with incorrect data in the image, I redirect it to the corrected image so anyone who tries to get there ends up at the correct image instead. For images that are removed and not replaced for whatever reason, let's create a media object (a JPG or whatever) that looks like this.

![image](https://github.com/ArctosDB/arctos/assets/5725767/97b574e8-1fa7-4c2a-88f6-4559a67ba187)

It could even be more generic and have the person wondering about the missing image file a Github issue.

We need to get more comfortable with the idea that you cannot fully delete anything (this is what a stable identifier means) but that we can tell anyone who cares that something has been deemed incorrect or outdated and send them on to better information. This doesn't make us dumb or sloppy - it means we are human (or an AI trained by humans...) or lack appropriate resources (or all of the above). We are not the only ones with this problem, but we could be the ones to "solve" it without just deleting stuff.

If the issue with a redirect is because the image can be accessed directly at TACC, bypassing Arctos - maybe we need to have a discussion about TACC providing this capability. I would think they would understand the need for stable (but ideally correct) data.

**Comment by dustymc** [comment]
>not getting something about redirects

They're tools - they don't do anything until someone uses them. (I'm not sure if they were used here or not, and we should address more than this specific use case.)

>cannot fully delete anything 

That would disallow users from doing things that others will see as 'Arctos is broken,' but IDK how usable it might be. 

>If the issue with a redirect is because the image can be accessed directly at TACC, bypassing Arctos

That's why https://handbook.arctosdb.org/documentation/media.html#urls-and-stability exists - that shouldn't be easy to get, and direct access should be violating licenses.

>TACC providing this capability

I suspect they'll see that as our job, but we could ask (if someone wants to write the proposal...) and some sort of sophistication there might make things like https://github.com/ArctosDB/internal/issues/240 more approachable.
